rules/windows/sysmon/sysmon_accessing_winapi_in_powershell_credentials_dumping.yml

```diff
@@ -0,0 +1,24 @@
+title: Accessing WinAPI in PowerShell. Credentials Dumping
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects Accessing to lsass.exe by Powershell
+status: experimental
+author: Natalia Shornikova
+date: 2020/10/06
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tag:
+  - attack.credential_access
+  - attack.t1003.001
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+      EventID:
+        - 8
+        - 10
+      SourceImage: '*\powershell.exe'
+      TargetImage: '*\lsass.exe'
```

Comment:
Could you please apply the`endswith ` [value modifier](https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#value-modifiers) here?

Reply:
Thank you! Done